lib/widgets/total_amount.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import '../l10n/tr.dart';
+
+class TotalAmount extends StatelessWidget {
```

Comment:
This widget should be under `src/ticket/widgets/total_amount.dart` because it's a local widget, not a global/reusable one.